src/webportal/src/app/job-submission/components/data/team-detail.jsx

```diff
@@ -353,4 +419,10 @@ export const SERVER_PATH = {
       {storage.data.path || '/'}
     </div>
   ),
+  dshuttle: storage => (
+    <div className={FontClassNames.semibold}>
+      <b>{`${storage.data.accountName}.blob.core.windows.net/${storage.data.containerName}`}</b>
+      {storage.data.path || '/'}
+    </div>
```

Comment:
Only return this when `storage.data.ufsType` is `wasb`. Else return `storage.data.ufsUri`?

Reply:
Done.